src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -22,27 +22,22 @@ export function useNxData(group: Group): NxData {
 
   const signalDataset = findSignalDataset(group);
   const signalValue = values[signalDataset.name];
-  if (!signalValue) {
-    return { signal: { dims: signalDataset.shape.dims } };
-  }
-
   assertArray<number>(signalValue);
-  const axesNames = getNxAxisNames(group);
 
-  const silxStyle = parseSilxStyleAttribute(group);
+  const silxStyle = getSilxStyle(group);
   const { axesScaleType, signalScaleType } = silxStyle;
 
-  const axisMapping = axesNames.map((axisName, i) => {
-    if (!axisName) {
+  const axisMapping = getNxAxes(group).map((axisName, i) => {
+    if (axisName === '.') {
```

Comment:
Checking for `'.'` here directly instead of converting them to `undefined` in `getNxAxes` (formerly `getNxAxisNames`).